### PR TITLE
docs: Created blog post for Uno beginners

### PR DESCRIPTION
**Pull request by ladyinblack**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Other... Please describe: Community Blog post for Advocacy Contribution

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

**For Hacktoberfest 2022**: Posted a blog, https://ronicasingh.hashnode.dev/introduction-to-uno-platform

Additional Comments: I have no idea if I'm doing this right, generally I know how to do pull requests, but the rules or instructions to follow to do a "PR" for an Advocacy Contribution is a little confusing.  I did go to the "Community Tutorials" link but I was not sure what I did was actually the correct procedure to follow.

Internal Issue (If applicable):

### CODE REVIEW

**Comment by sasakrsmanovic** [comment]
Thanks for your contribution Ronica. Yes you did this right, and you've placed your blog post at the right hacktoberfest community advocaacy section. I read over the blog - great to see you are excited about Uno Platform and I look forward to your future blogs!

**Review by sasakrsmanovic — APPROVED** [review]
great intro blog; thanks

**Comment by amelield** [comment]
I just read your blog post, and I really liked it. You make it very accessible to neophytes. You got a new reader. Make sure to use conventional commit form https://www.conventionalcommits.org/en/v1.0.0/#summary

File: doc/articles/guides/community-tutorials.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ This page lists links our community provided tutorials.
 - [Uno Platform Getting Started with New Projects](https://www.andrewhoefling.com/Blog/Post/uno-platform-getting-started-with-new-projects), by Skye Hoefling
 - [Uno Platform - Build a Todo App - Part 5](https://steven-giesel.com/blogPost/2c025ac6-d67f-45ec-a616-009e0285c999), by Steven Giesel
 - [Uno Platform on Surface Duo](https://devblogs.microsoft.com/surface-duo/tag/uno-platform/), by Craig Dunn
-- _Create a PR tp add yours here!_
\ No newline at end of file
+- [Introduction to Uno Platform](https://ronicasingh.hashnode.dev/introduction-to-uno-platform), by Ronica Singh
+- _Create a PR tp add yours here!_
```
Comment (Xiaoy312) [inline doc/articles/guides/community-tutorials.md:14]: since we are here, might as well:
```suggestion
- _Create a PR to add yours here!_
```

**Comment by MartinZikmund** [comment]
@ladyinblack you need to make sure the commit message is changed to "docs: ____" to pass the conventional commits validation. Please search for git reword to find info on how to reword the last commit and then force-push to overwrite it. Let us know if you need some additional pointers how to do this 👍

**Comment by sasakrsmanovic** [comment]
Ping @ladyinblack - Hacktoberfest is almost over. We'd love to get this one in. Let us know if you need more help in getting it resolved. Thanks again for your contribution.

**Comment by ladyinblack** [comment]
@sasakrsmanovic @MartinZikmund 

Hello, I'm not sure what or how to do this.  I've added the "docs: " before, but I'm unsure how to resolve these four failed checks.

**Comment by MartinZikmund** [comment]
> @sasakrsmanovic @MartinZikmund 
> 
> Hello, I'm not sure what or how to do this.  I've added the "docs: " before, but I'm unsure how to resolve these four failed checks.
> 

@ladyinblack Hi, you have changed the PR title only, but not the _commit_ message:



As mentioned above - this can be done using git reword, and then by force-pushing. See this tutorial for the steps - https://linuxize.com/post/change-git-commit-message/

**Comment by MartinZikmund** [comment]
@ladyinblack adjusted the commit message, so this will soon be merged

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@ladyinblack can you please confirm the CLA by clicking the button in the previous message? This is needed so the commit can be merged. Thanks!
image

**Pull request merged**